converter/src/DelphesEDM4HepConverter.cc

```diff
@@ -435,11 +435,9 @@ namespace k4SimDelphes {
   }
 
   void DelphesEDM4HepConverter::processScalarHT(const TClonesArray* delphesCollection, std::string const& branch) {
-    auto* collection  = createCollection<edm4hep::ParticleIDCollection>(branch);
+    auto* collection  = createCollection<podio::UserDataCollection<float>>(branch);
```

Comment:
I assume a `float` is enough here and we don't want / need a `double`?

Reply:
I followed what we did for the IsolationVariable where we had a float, quite sure that it's ok for precision.